packages/phoenix-evals/pyproject.toml

```diff
@@ -22,10 +22,38 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
+  "openinference-semantic-conventions",
+  "pandas",
+  "tqdm",
+  "typing-extensions>=4.5, <5",
 ]
 
 [project.optional-dependencies]
+dev = [
+  "anthropic",
+  "boto3",
+  "litellm>=1.0.3",
+  "openai>=1.0.0",
+  "tiktoken",
```

Comment:
didn't we deprecate? Or is that post?

Reply:
We should also ticket the need to remove this

Reply:
Tracked by #2357